functions/Install-DbaSqlWatch.ps1

```diff
@@ -0,0 +1,231 @@
+#ValidationTags#CodeStyle,Messaging,FlowControl,Pipeline#
+function Install-DbaSqlWatch {
+    <#
+    .SYNOPSIS
```

Comment:
fyi, i committed a few changes - one of which was a small formatting change to tab the help over to the left

Reply:
Got it. Thank you!